viewer/vue-client/src/resources/json/structuredValueTemplates.json

```diff
@@ -505,6 +505,18 @@
     "@type": "BiographicalOrHistoricalData",
     "label": ""
   },
+  "FieldOfActivity": {
+    "@type": "FieldOfActivity",
+    "label": ""
+  },
+  "Occupation": {
+    "@type": "Occupation",
+    "label": ""
+  },
+  "marc:OtherAttributes": {
+    "@type": "marc:OtherAttributes",
+    "label": ""
```

Comment:
Just verifying: We don´t have to think of converting to marc? No label in marcframe.